ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -336,46 +397,93 @@ const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById })
   const quality = Number.isInteger(genotype.gq) ? genotype.gq : genotype.qs
   const filters = genotype.filters?.join(', ') || variant.genotypeFilters
 
+  return {
+    genotype,
+    isHemiX,
+    warning,
+    previousCall,
+    hasConflictingNumAlt,
+    details,
+    showSecondaryQuality,
+    secondaryQuality,
+    quality,
+    filters,
+  }
+}
+
+const Genotype = React.memo(({ variant, individual, isCompoundHet, genesById }) => {
+  if (!variant.genotypes) {
+    return null
+  }
+
+  let genotypes = variant.genotypes[individual.individualGuid]
```

Comment:
in general, using `let` and then changing the value as you go is more bug prone than using `const`. Unless there is a really compelling reason why you need to change the value it is better to define new constants -
for example: 
```
const individualGenotypes = variant.genotypes[individual.individualGuid]
const genotypes =  (Array.isArray(individualGenotypes) ? individualGenotypes : [individualGenotypes]).sort(
  ...).map(...)
```